deployments/cloudfoundry/bosh/config/blobs.yml

```diff
@@ -0,0 +1,6 @@
+signalfx_agent/signalfx-agent.tar.gz:
+  size: 194032255
+  sha: sha256:40ca24994f792d4b7c40f8640bea1fc420fbfe5eed918b4d63b4293e3b1e07dd
+splunk_otel_collector/splunk_otel_collector:
+  size: 163463346
+  sha: sha256:db0ddd022c2b8f4fa00d3d8f9764fcee4cc1de9fb2da32b1f7d536b677c89261
```

Comment:
I'm not familiar with bosh.  Are these sizes and checksums used to validate the downloaded packages?  Does this file need to updated when the package versions are changed?

Reply:
Yes, they're sizes and checksums of the blobs (versioned files essentially in this case). These get updated by the `bosh add-blob` command, which is run by the release script, so there shouldn't be a need for the repo itself to be updated when package versions are changed.